src/Node.hs

```diff
@@ -77,14 +77,16 @@ data ListenerAction packing m where
   -- | A listener that handles an incoming bi-directional conversation.
   ListenerActionConversation
     :: ( Packable packing snd, Unpackable packing rcv, Message rcv )
-    => (LL.NodeId -> ConversationActions snd rcv m -> m ())
```

Comment:
Suppose we exchanged some data and then want to distribute it. Though probably this should be done with workers... We had one case actually, but then it was reworked to worker :)